de.metas.report/report-service/src/main/java/de/metas/report/rest/ReportRestController.java

```diff
@@ -119,7 +97,7 @@ private static JsonReportError toJsonReportError(final Throwable throwable, fina
 	{
 		final Throwable cause = AdempiereException.extractCause(throwable);
 
-		final String adLanguageEffective = Check.isNotBlank(adLanguage)
+		final String adLanguageEffective = !Check.isBlank(adLanguage)
```

Comment:
why?

Reply:
this somehow got here when merging the old branch into this one1